.github/workflows/test-e2e-junk.yaml

```diff
@@ -0,0 +1,92 @@
+name: TestSuite E2E Junk
+
+on:
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+    inputs:
+      k8s_version:
+        description: "With Kubernetes version"
+        required: false
+
+jobs:
+  wait-for-image-build:
+    name: "Wait for image build"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/wait-for-image-build
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          statusName: pull-lifecycle-mgr-build
```

Comment:
comment: these here used to be a conditional based on whether the job is running for a `main` build or a `PR` build. Since we removed the post `main` build, I removed this conditional now.